sky/templates/aws-ray.yml.j2

```diff
@@ -185,7 +186,12 @@ setup_commands:
     sudo grep -e '^DefaultTasksMax' /etc/systemd/system.conf || (sudo bash -c 'echo "DefaultTasksMax=infinity" >> /etc/systemd/system.conf'); sudo systemctl set-property user-$(id -u $(whoami)).slice TasksMax=infinity; sudo systemctl daemon-reload;
     {%- endif %}
     mkdir -p ~/.ssh; (grep -Pzo -q "Host \*\n  StrictHostKeyChecking no" ~/.ssh/config) || printf "Host *\n  StrictHostKeyChecking no\n" >> ~/.ssh/config;
-    [ -f /etc/fuse.conf ] && sudo sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf || (sudo sh -c 'echo "user_allow_other" > /etc/fuse.conf'); # This is needed for `-o allow_other` option for `goofys`;
+    [ -f /etc/fuse.conf ] && sudo sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf || (sudo sh -c 'echo "user_allow_other" > /etc/fuse.conf');
+    {%- if remote_identity != 'LOCAL_CREDENTIALS' %}
+    rm ~/.aws/credentials || true;
```

Comment:
Say a user manually logged in a configured AWS with `aws configure`. Would a new `sky launch` then delete their custom added credentials?

Reply:
Yes, we will delete their custom-added credentials. A user setting up `aws configure` themselves on the remote cluster can be an undefined behavior, as that could break our autostop as well.

A better solution is to make the credentials declarative, i.e. note the credential/active service account for SkyPilot in the cluster yaml and use the credential everywhere in our backend. This is a temp solution to unblock our users. Wdyt?

Reply:
sgtm!